opam-health-check.opam

```diff
@@ -46,8 +46,8 @@ depends: [
   "lwt"
   "lwt_ppx"
   "uri"
-  "docker_hub" {>= "0.1.0"}
-  "memtrace"
+  "docker_hub" {>= "0.1.0" & < "0.2.0"}
```

Comment:
Should we add a TODO noting that we'd like to make updates to use the latest version when feasible? Or is there a reason that we would not plan to do that?

Reply:
The new version is a complete API change, and we have no pressing need do incorporate it.